### PR TITLE
Make CASSANDRA_VERSION a Version object

### PR DESCRIPTION
**Pull request by bjmb**

### CODE REVIEW

File: tests/integration/__init__.py [hunk 1]
```diff
@@ -219,20 +217,20 @@ def get_unsupported_upper_protocol():
...
-greaterthancass20 = unittest.skipUnless(CASSANDRA_VERSION >= '2.1', 'Cassandra version 2.1 or greater required')
-greaterthancass21 = unittest.skipUnless(CASSANDRA_VERSION >= '2.2', 'Cassandra version 2.2 or greater required')
-greaterthanorequalcass30 = unittest.skipUnless(CASSANDRA_VERSION >= '3.0', 'Cassandra version 3.0 or greater required')
-greaterthanorequalcass36 = unittest.skipUnless(CASSANDRA_VERSION >= '3.6', 'Cassandra version 3.6 or greater required')
-greaterthanorequalcass3_10 = unittest.skipUnless(CASSANDRA_VERSION >= '3.10', 'Cassandra version 3.10 or greater required')
-greaterthanorequalcass3_11 = unittest.skipUnless(CASSANDRA_VERSION >= '3.11', 'Cassandra version 3.10 or greater required')
-greaterthanorequalcass40 = unittest.skipUnless(CASSANDRA_VERSION >= '4.0', 'Cassandra version 4.0 or greater required')
-lessthanorequalcass40 = unittest.skipIf(CASSANDRA_VERSION >= '4.0', 'Cassandra version 4.0 or greater required')
-lessthancass30 = unittest.skipUnless(CASSANDRA_VERSION < '3.0', 'Cassandra version less then 3.0 required')
+greaterthancass20 = unittest.skipUnless(CASSANDRA_VERSION >= Version('2.1'), 'Cassandra version 2.1 or greater required')
```
Comment (mambocab) [inline tests/integration/__init__.py]: Just a heads up: this is fine, but not necessary -- a `Version` will "cast" the string operand to a `Version` on comparison:

https://github.com/python/cpython/blob/master/Lib/distutils/version.py#L168

**Comment by bjmb** [comment]
Yeah, good idea, I'll run the whole matrix on this branch and compare it manually with master.

**Comment by bjmb** [comment]
Looks good @mambocab , in fact we are running one more test now since `tests.integration.standard.test_udts.UDTTests.test_non_frozen_udts` was being skipped before because `Cassandra version 3.6 or greater required`. This doesn't happen anymore with the new comparison.

**Comment by mambocab** [comment]
Awesome, thanks for checking -- +1 to merging!

**Pull request merged**